src/Classes/Models/StageModel.php

```diff
@@ -10,4 +12,37 @@ public function __construct(\PDO $db)
     {
         $this->db = $db;
     }
+
+    /**
+     * Sets the 'deleted' flag to '1' for a record with a given id.
+     *
+     * @param integer $id
+     *
+     * @return boolean for success or failure of the query
+     */
+    public function deleteStage(int $id) : bool
+    {
+        $query = $this->db->prepare("UPDATE `stages` SET `deleted` = '1', `order` = '0' WHERE `id` = :id");
+        $query->bindParam(':id', $id);
+        return $query->execute();
+    }
+
+    /**
+     * Retrieves a stage with the specified id
+     *
+     * @param integer $id
+     * @return StageEntity object
+     *
+     */
+    public function getStageById(int $id) : StageEntity
+    {
+        $query = $this
+        ->db
+        ->prepare('SELECT `stages`.`id`, `title`, `order`, `deleted` FROM `stages` WHERE `stages`.`id`=:id');
```

Comment:
Could remove `stages`. from the beginning of the Select and Where sections to shorten your sql statement